omod/src/main/java/org/openmrs/module/fhir2/web/servlet/FhirRestServlet.java

```diff
@@ -108,4 +112,9 @@ public void setResourceProviders(Collection<IResourceProvider> theProviders) {
 	public void setServerAddressStrategy(IServerAddressStrategy theServerAddressStrategy) {
 		super.setServerAddressStrategy(theServerAddressStrategy);
 	}
+	
+	@Autowired
+	public void setMessageSource(MessageSource messageSource) {
```

Comment:
Used a setter so that `ReloadableResourceBundleMessageSource` could be set as `messageSource` in web tests.